bitnami/mongodb/templates/standalone/pvc.yaml

```diff
@@ -27,4 +27,7 @@ spec:
     requests:
       storage: {{ .Values.persistence.size | quote }}
   {{ include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) }}
+  {{- if .Values.persistence.volumeClaimTemplates.selector }}
+  selector: {{- include "common.tplvalues.render" (dict "value" .Values.persistence.volumeClaimTemplates.selector "context" $) | nindent 4 }}
+  {{- end -}}
```

Comment:
IMHO this parameter should be renamed to `.Values.persistence.selector` and be applied both here and in the statefulset's  volumeClaimTemplate.

Reply:
hm.. but `Values.yaml` for mongodb chart don't have this section -- it's all inside volumeClaimTemplates subsection.
does it mean that we need to add extra field in `Values.yaml` as well?